style.css

```diff
@@ -0,0 +1,87 @@
+@font-face {
+    font-family: "Comic Sans MS";
+    src: url(/assets/fonts/ComicSansMS.ttf);
+}
+
+:root {
+    --background-color: #FFFFFF;
+    --text-color: #000000;
+}
+
+* {
+    -webkit-user-drag: none;
+    user-select: none;
+    -moz-user-select: none;
+    -webkit-user-select: none;
+    -ms-user-select: none;
+}
+
+body {
+    font-family: "Comic Sans MS";
+    background: var(--background-color);
+    color: var(--text-color);
+}
+
+@media (prefers-color-scheme: dark) {
+    :root {
+        --background-color: #000000;
+        --text-color: #FFFFFF;
+    }
+}
+
+#roingus {
+    box-sizing: border-box;
+    position: absolute;
+    scale: 1.5;
+    transform-origin: center center;
+    transition: 300ms cubic-bezier(0.075, 0.82, 0.165, 1);
+}
+
+#roingus:hover {
+    cursor: url(/assets/images/reaching.png) 16 16, grab;
+}
+
+#roingus:active {
+    cursor: url(/assets/images/grab.png) 16 16, grabbing;
+    scale: 1.2;
+}
+
+.center {
```

Comment:
If you use `.main-container` to layout the content, this class would be kinda redundant. (see the comment below)